src/module_2/module_2_1.md

```diff
@@ -0,0 +1,402 @@
+```python
+import numpy as np
+import pandas as pd
+import os
+
+
+folder_path = "../../data" #Folder in main directory
+
+# # Crea el directorio local si no existe
+# if not os.path.exists(data_folder):
+#     os.makedirs(data_folder)
+```
+
+
+```python
+# Clean Users dataframe
+# Importing dataframe from "dirty" folder
+df = pd.read_parquet(f"{folder_path}/users.parquet")
+
+# Drop information about users with no adults
+drop_indexes = df[
+    ((df["count_adults"] == 0) & (df["count_people"] > 0))
```

Comment:
cleaner if yo do the sum of those columns on each row `df[df[the_4_cols].sum() > 0]`